server/admin_management/api/app/schemas.py

```diff
@@ -96,3 +152,41 @@ class FamAppAdminGet(BaseModel):
     application: FamApplicationBase
 
     model_config = ConfigDict(from_attributes=True)
+
+
+# -------------------------- FAM Access Control Privilege (Delegated Admin) -------------------------- #
+class FamAccessControlPrivilegeCreateRequest(BaseModel):
+    user_name: Annotated[str, StringConstraints(min_length=3, max_length=20)]
+    user_type_code: famConstants.UserType
+    role_id: int
+    forest_client_number: Union[
+        List[Annotated[str, StringConstraints(min_length=1, max_length=8)]], None
+    ] = None
+
+    model_config = ConfigDict(from_attributes=True)
+
+
+class FamAccessControlPrivilegeCreate(BaseModel):
```

Comment:
Hi Catherine, this `FamAccessControlPrivilegeCreate` looks confusing with `FamAccessControlPrivilegeCreateRequest`. By the name from them I can't tell what their purpose are for which endpoints. Normally `schema.py` is where endpoints' request body classes and response body classes located.

It also looks a bit strange as "user_id", "create_user" are in its properties. I would not assume frontend at this point pass them in request body for granting access. Maybe this is for some other purpose?

Reply:
renamed to FamAccessControlPrivilegeCreateDto, and added comment to both